cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/TransactionHttpHandler.java

```diff
@@ -175,4 +208,78 @@ public void resetTxManagerState(HttpRequest request, HttpResponder responder) {
     txClient.resetState();
     responder.sendStatus(HttpResponseStatus.OK);
   }
+
+  @Path("/transactions/pruneinfo")
```

Comment:
As we have another endpoint (`live-info`), that would be consistent and readable.